x-pack/plugins/actions/server/lib/mustache_lambdas.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as tinymath from '@kbn/tinymath';
+import { parse as hjsonParse } from 'hjson';
+
+import moment from 'moment-timezone';
+
+import { Variables } from './mustache_renderer';
+
+const DefaultDateTimeZone = 'UTC';
+const DefaultDateFormat = 'YYYY-MM-DD hh:mma';
+
+export function getMustacheLambdas(): Variables {
+  return getLambdas();
+}
+
+const TimeZoneSet = new Set(moment.tz.names());
+
+type RenderFn = (text: string) => string;
+
+function getLambdas() {
+  return {
+    EvalMath: () =>
+      function (this: Variables, text: string, render: RenderFn) {
```

Comment:
Not an option.  The way these lambdas are set up by mustache, is that they call the function with `this` set to the "variables" available to the lambda.  We **HAVE** to define it this way.

Worth a comment though, I think, because it's obviously a little confusing, especially using `this` as a "function variable" like this (it's a special typescript thing).  see: https://www.typescriptlang.org/docs/handbook/2/functions.html#declaring-this-in-a-function